### PR TITLE
Update Gitpod ports

### PR DESCRIPTION
**Pull request by tyler36**
## The Issue
I was debugging Mailpit issues in Gitpod when I noticed many of the ports are outdated.

## How This PR Solves The Issue
This PR 

- updates the ports to match those from
https://github.com/ddev/ddev/blob/master/.gitpod.yml
- adds postgres port

This PR does NOT update the Mailpit port. See #40

### CODE REVIEW

**Comment by tyler36** [comment]
OK. Not sure what changes you would like here though.

**Comment by rfay** [comment]
There is no way to change it here, as it needs to be changed in DDEV. 

But I'm not sure why it needs to be changed. Is it `ddev mailpit` that needs to be changed? What is the goal?

**Comment by tyler36** [comment]
Consistency, mostly. 

EG. Xdebug port:

- FIle lists `9000` as xdebug port, but xdebug is on `9003`. So this is confusing.
- If it doesn't matter (ie, we don't use it), lets delete the entry so there's less maintainance.

**Comment by rfay** [comment]
Xdebug has been on 9003 for several years, since before gitpod support. I'm wondering how it got that?

If you can live with mailpit listening on 8027, it's OK there. People shouldn't have to know where these ports are.

**Comment by rfay** [comment]
Sorry, I got confused between your two PRs

**Comment by tyler36** [comment]
So where are we with this?

**Comment by stasadev** [comment]
I think it's good.

Can you also add more ports from the official add-ons?:

- https://github.com/ddev/ddev-adminer
- https://github.com/ddev/ddev-xhgui

**Comment by tyler36** [comment]
Added ports but have not tested.

File: .gitpod.yml [hunk 1]
```diff
@@ -46,14 +45,17 @@ ports:
...
     onOpen: ignore
-  # router http port that we're ignoring.
-  - port: 8888
+  # Direct-connect ddev-webserver port that is the main port
+  - port: 8080
     onOpen: ignore
-  # router https port that we're ignoring.
-  - port: 8889
+  # xhgui port
+  - port: 8142
```
Comment (stasadev) [inline .gitpod.yml:52]:
```suggestion
  # xhgui http port
  - port: 8143
    onOpen: ignore
  # xhgui https port
  - port: 8142
```

File: .gitpod.yml [hunk 2]
```diff
@@ -46,14 +45,17 @@ ports:
...
     onOpen: ignore
   # xdebug port
-  - port: 9000
+  - port: 9003
+    onOpen: ignore
+  # Adminer
+  - port: 9101
```
Comment (stasadev) [inline .gitpod.yml:58]:
```suggestion
  # Adminer http port
  - port: 9100
    onOpen: ignore
  # Adminer https port
  - port: 9101
```

**Review by stasadev — CHANGES_REQUESTED** [review]
Thanks, I see you added HTTPS ports, but we also need HTTP, right?
For `docker-compose.*_norouter.yaml` that uses HTTP.

**Comment by tyler36** [comment]
@stasadev Oops. Forgot the HTTP ports. Good catch!

**Review by stasadev — APPROVED** [review]
Looks good to me.

**Pull request merged**